Microsoft.Toolkit.Uwp.UI.Controls.Graph/PowerBIEmbedded/Embed/EmbedReportFrame.html

```diff
@@ -0,0 +1,246 @@
+﻿<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title></title>
+    <style>
+        html, body {
+            border: 0px;
+            margin: 0px;
+            height: 100%;
+            font-family: 'Segoe UI Web Light', 'Segoe UI Light', 'Segoe WP Light', 'Segoe UI', 'Segoe WP', Tahoma, Arial, sans-serif;
+            overflow: hidden;
+        }
+        #dropdownContainer {
+            position: absolute;
+            left: 10px;
+            top: 10px;
+        }
+        #dropdownContainer select {
+            margin-right: 15px;
+            padding: 3px;
+            font-size: 1em;
+            display: inline-block;
+        }
+        #reportContainer {
+            height: 100%;
+        }
+
+        @media (max-width: 720px) {
+            #dropdownContainer {
+                position: static;
+            }
+            #dropdownContainer select {
+                width: 100%;
+                display: block;
+            }
+        }
+    </style>
+    <script src="powerbi.min.js"></script>
+</head>
+<body>
+    <div id="dropdownContainer" style="visibility:hidden">
```

Comment:
Looks like there should be padding below these:

![image](https://user-images.githubusercontent.com/24302614/42787600-04e0de72-8910-11e8-808e-d1f29e20e3f9.png)

Though, would it be hard to move these to the XAML template instead?  Then the control could be restyled easily to remove them if the developer only has one specific report to show?

Reply:
@michael-hawker There was some limitations to do that in XAML, so we decided to render the dropdowns in HTML, I just updated the CSS to make it better.

Reply:
The dropdowns will be rendered inline now